notes/returned-mail.vbs

```diff
@@ -166,39 +166,49 @@ IF ADDR_actions <> "no response received" THEN
     	mailing_address_confirmed = "YES"
     End If
     Dialog1 = "" 'Blanking out previous dialog detail
-    BeginDialog Dialog1, 0, 0, 566, 155, "RETURNED MAIL PROCESSING"
+    BeginDialog Dialog1, 0, 0, 566, 155, "RETURNED MAIL "
       DropListBox 220, 15, 55, 15, "Select:"+chr(9)+"YES"+chr(9)+"NO", residential_address_confirmed
       DropListBox 500, 15, 55, 15, "Select:"+chr(9)+"YES"+chr(9)+"NO", mailing_address_confirmed
-      EditBox 85, 75, 195, 15, returned_mail
-      EditBox 85, 95, 195, 15, verifications_requested
-      EditBox 85, 115, 195, 15, other_notes
-      EditBox 85, 135, 45, 15, due_date
-      EditBox 365, 115, 195, 15, notes_on_address
+      EditBox 90, 75, 190, 15, returned_mail
       Text 10, 25, 200, 25, resi_addr_street_full
       Text 10, 35, 210, 15, resi_addr_city &  ", "  & resi_addr_state & " "   & resi_addr_zip
       Text 290, 25, 200, 25, mail_street_full
-      If mail_city <> "_______________" Then Text 290, 35, 200, 10, mail_city_line & ", "  & mail_state_line &  " "  & mail_zip_line
+      If mail_city_line <> "_______________" THEN Text 290, 35, 200, 10, mail_city_line & ", "  & mail_state_line &  " "  & mail_zip_line
```

Comment:
Line 176 - mail_city_line is "" so this condition will never be met. See function access_ADDR_panel for functionality. This is why the comma is still there.